chapters/background.tex

```diff
@@ -1,4 +1,14 @@
 \section{Background} \label{sec:background}
 % identifying existing approaches within the research literature; this may potentially include mobile-based and non-mobile approaches if necessary
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-
+Regarding soHappy, TesnorFlow \cite{tensorflow} appears to be a convenient fit when it comes to smile detection.
+TensorFlow is a open source machine learning library developed by Google.
+It supports a wide variety of programming languages, mainly targeting web and mobile applications.
+For mobile development TensorFlow provides a Lite version called TensorFlow Lite (TFLite) which can be easily included in a mobile app.
+The main advantage TFLite offers, is the ability to add a self trained model to detect smiles.
```

Comment:
The comma is probably unnecessary here. Not sure whether it's actually _wrong_ though.